contracts/AllowanceModule.sol

```diff
@@ -238,21 +238,21 @@ contract AllowanceModule is SignatureDecoder, Ownable {
              // solium-disable-next-line
             emit PayAllowanceTransfer(address(safe), delegate, paymentToken, GELATO, payment);
             }
-              // Transfer token
+        
+        // Transfer token
         transfer(safe, token, to, amount);
         emit ExecuteAllowanceTransfer(address(safe), delegate, token, to, amount, allowance.nonce - 1);
     }
 
     function createGelatoTask(address token, address paymentToken) external {
-        require(paymentTokens[msg.sender][paymentToken], "payment token not whitelisted");
+        if (paymentToken != ETH) require(paymentTokens[msg.sender][paymentToken], "payment token not whitelisted");
```

Comment:
The exec function is written in such a way to not require ETH to be whitelisted on the Allowance Module as a payment token